src/styles/ForecastBreakdown.css

```diff
@@ -0,0 +1,40 @@
+.forecast-breakdown {
+  display: flex;
+  flex-direction: row-reverse;
+  justify-content: space-between;
+  flex: 1 1 auto;
+  text-align: center;
+  min-height: 350px;
+  padding: 20px;
+}
+
+.forecast-breakdown__left {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+}
+
+.forecast-breakdown__left h3, .forecast-breakdown__left h3 + p {
+  display: inline;
+}
+
+.forecast-breakdown__left > div > * {
```

Comment:
Be wary of making your selectors overly specific. Shoul you need to change something later on it can be hard to overwrite things like this without unintended consequences, especially is they are using general elements like a div as a target. Its much better to add a classname to the element you are trying to hit and avoid the over specificity youre using here